pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -137,7 +138,10 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 		return fmt.Errorf("couldn't create deployment for deployment config %s: %v", deployutil.LabelForDeploymentConfig(config), err)
 	}
 	c.recorder.Eventf(config, "DeploymentCreated", "Created new deployment %q for version %d", created.Name, config.Status.LatestVersion)
-	return nil
+	// Update deploymentConfig status; we probably need a separate call for status
```

Comment:
Opened https://github.com/openshift/origin/issues/6465 for a separate status update call.